pandas/core/generic.py

```diff
@@ -8976,7 +8955,6 @@ def resample(
             freq=rule,
             label=label,
             closed=closed,
-            kind=kind,
```

Comment:
i dont see a diff in pandas.core.resample; doesn't the keyword need to be removed there too?

Reply:
sorry, my mistake. I removed the keyword `kind` from `get_resampler`

Reply:
Shouldn't all/most of the `kind` keywords in `resample.py` be removable?

Reply:
Thanks for the comment. Yes, it was my mistake; `kind` can be removed from `Resampler` and `TimeGrouper`. I have updated this PR. Could you please take a look?